src/client/src/apps/MainRoute/widgets/spotTile/components/stories/AnalyticsTitle.stories.tsx

```diff
@@ -18,7 +30,34 @@ const rfqActions = {
   reset: action('reset'),
 }
 
-stories.add('AnalyticsTile', () => (
+stories.add('AnalyticsTile', () => {
```

Comment:
Now that the number of stories for subcomponents `Spot Tile` has grown quite large, I think it would be a nice idea to add a clearer hierarchy: eg

```diff
--- a/src/client/src/apps/MainRoute/widgets/spotTile/components/stories/AnalyticsTitle.stories.tsx
+++ b/src/client/src/apps/MainRoute/widgets/spotTile/components/stories/AnalyticsTitle.stories.tsx
@@ -1,9 +1,10 @@
 import React from 'react'
-import { stories, Story, Centered } from './Initialise.stories'
+import { Story, Centered } from './Initialise.stories'
 import { action } from '@storybook/addon-actions'
 import { ServiceConnectionStatus } from 'rt-types'
 import { currencyPair, spotTileData } from '../test-resources/spotTileProps'
 import { AnalyticsTile } from '../analyticsTile'
+import { storiesOf } from '@storybook/react'

 const noPriceData = {
   ...spotTileData,
@@ -30,7 +31,9 @@ const rfqActions = {
   reset: action('reset'),
 }

-stories.add('AnalyticsTile', () => {
+const stories = storiesOf('Spot Tile.AnalyticsTile', module)
+
+stories.add('Default', () => {
   return (
     <Story>
       <Centered>
   return (
     <Story>
       <Centered>
@@ -57,7 +60,7 @@ stories.add('AnalyticsTile', () => {
   )
 })

-stories.add('AnalyticsTile in error', () => (
+stories.add('Error', () => (
   <Story>
     <Centered>
       <div
@@ -82,7 +85,7 @@ stories.add('AnalyticsTile in error', () => (
   </Story>
 ))

-stories.add('AnalyticsTile no price', () => (
+stories.add('No price', () => (
   <Story>
     <Centered>
       <div
```

Which results in:

<img width="220" alt="Screenshot 2020-01-24 at 11 06 27" src="https://user-images.githubusercontent.com/156303/73064629-97727300-3e99-11ea-8171-c5191b8932ef.png">

Reply:
Good point! will do a nice structure

Reply:
Good point! will do a nice structure